README.md

```diff
@@ -109,6 +133,11 @@ Add `C:\msys64\mingw64\bin` to your `PATH` environment variable.
 echo "???"
 ```
 
+##### Zig (Windows)
+```bash
+echo "???"
+```
```

Comment:
I don't really care about supporting the individual workflows for Windows, as we will likely just embed zig soon